src/template-tags/tickets.php

```diff
@@ -707,6 +707,32 @@ function tribe_tickets_has_meta_fields( $id, $context = null ) {
 }
 
 /**
+ * Updates a given Object Capacity
+ *
+ * @since  TBD
+ *
+ * @param  int  $object  Post We are trying to save capacity
+ *
+ * @return int|false
+ */
+function tribe_tickets_update_capacity( $object, $capacity ) {
+	if ( ! $object instanceof WP_Post ) {
+		$object = get_post( $object );
+	}
+
+	if ( ! $object instanceof WP_Post ) {
+		return false;
+	}
+
+	if ( ! is_numeric( $capacity ) ) {
```

Comment:
This is fine, but it could really move up before `if ( ! $object instanceof WP_Post )` since it doesn't depend on it? Then we skip the double testing altogether.